src/utils/renderBatch.js

```diff
@@ -17,7 +17,7 @@ export default (config, isClosing) => (req, res) => {
       if (isClosing()) {
         logger.info('Ending request when closing!');
       }
-      res.status(manager.statusCode).json(manager.getResults()).end();
+      return res.status(manager.statusCode).json(manager.getResults()).end();
```

Comment:
Here's what the method does:

https://nodejs.org/api/http.html#http_response_end_data_encoding_callback